src/analysis/ContextBar.ts

```diff
@@ -225,14 +275,16 @@ export const ContextBar = ({
                   'below.',
                 ]),
                 br({ key: 'br' }),
-                div({ key: 'p2' }, 'Workflow and workspace storage costs\nare not included.'),
+                div({ key: 'p2' }, ['Workflow and workspace storage costs\nare not included.']),
               ],
             },
             [
               h(
                 Interactive,
                 {
                   as: 'div',
+                  // TODO: debug
+                  // @ts-expect-error
```

Comment:
I'm fairly sure I should be changing `Interactive.d.ts` to allow a `style` element since it is indeed valid, but I spent a bit too much time on it. Any insight @nawatts ?

Reply:
Casting the props to `InteractiveProps<'div'>` seems to work.

On a related note, I think we'll probably end up switching to a simpler props type for Interactive without a type parameter (#3992 copies AoU's version of Interactive).